src/contact.js

```diff
@@ -0,0 +1,26 @@
+import anchorTag from './anchorTag';
+
+const contact = () => {
+  const content = document.querySelector('div#content');
+  content.innerHTML = '';
+  const div = document.createElement('div');
+  div.setAttribute('class', 'text-white jumbotron contact align-items-center min-vh-100');
+  const p = document.createElement('p');
+  p.setAttribute('class', 'container text-center');
+  p.innerText = 'Contact Us Using the Social Media';
+  div.appendChild(p);
+  const brakLine = document.createElement('br');
+
+  const nav = document.createElement('ul');
+  nav.setAttribute('class', 'nav nav-tabs justify-content-center');
+
+  nav.appendChild(anchorTag('Instagram', 'fa fa-instagram'));
```

Comment:
- [ ] **[OPTIONAL]** We can increase the speed and performance of appending children to our nodes by avoiding reflows (drawing the page every time you append a child to a certain div) using [`createDocumentFragment`](https://developer.mozilla.org/en-US/docs/Web/API/Document/createDocumentFragment) function. 

Think of it as a temporary element created in memory that you append children instead of the target element, and once you are done with appending any number of children, you append this fragment to the target without affecting the performance (one reflow vs multiple reflows). We can apply the same concept in the other modules.